R/utility.R

```diff
@@ -1,35 +1,46 @@
-getInstallationCommand <- function(packages){
-  installation <- ""
-
-  for(package in packages){
-    installation <- paste0(installation,
-                         sprintf("Rscript -e \'args <- commandArgs(TRUE)\' -e \'install.packages(args[1], dependencies=TRUE)\' %s", package),
-                         ";")
+getJobPackageInstallationCommand <- function(type, packages){
+  script <- ""
+  if (type == "cran") {
+    script <- "Rscript $AZ_BATCH_JOB_PREP_WORKING_DIR/install_cran.R"
+  }
+  else if (type == "github") {
```

Comment:
If the user wants to specify both, the user would need to do each function individually..
getJobPackageInstallationCommand("cran", c("httr", "ludridate"))
getJobPackageInstallationCommand("cran", c("Azure/doAzureParallel", "Azure/rAzureBatch"))

Should the function actually do both? This is internal function

getJobPackageInstallationCommand(cran = c("ludridate", "httr"), github = c("Azure/doAzureParallel", "Azure/rAzureBatch"))